docs/insomnia/context-object-reference.md

```diff
@@ -33,11 +33,11 @@ interface RequestContext {
     removeParameter(name: string): void;
     getBody(): Object;
     setBody(body: Object): void;
-    setCookie(name: string, value: string): void;
     getEnvironmentVariable(name: string): any;
     getEnvironment(): Object;
     setAuthenticationParameter(name: string, value: string): void;
     getAuthentication(): Object;
+    getCookie(name: string): string | null;
```

Comment:
```suggestion
```
Good catch on the double. However, we don't have `getCookie` available. Would you delete this line?